src/modules/api.js

```diff
@@ -38,11 +42,43 @@ export default class Api {
     return data.error ? [] : data;
   };
 
+  likeUrl =() => `${this.APP_URL}${this.APP_ID}/likes/`
+
+  getLikes = async () => {
+    const response = await fetch(this.likeUrl());
+    if ((response.status && response.status === 500) || response.status === 400) {
+      return [];
+    }
+
+    const data = await response.json();
+    return data.error ? [] : data;
+  };
```

Comment:
There is another function that is similar to this. What do you think if you refactor them? 
[api.js#L32-L39](https://github.com/matovu-farid/pokemon/blob/a114ca39e338c044787abe72c1b8e23721ae44d3/src/modules/api.js#L32-L39)